jwst/resample/resample_utils.py

```diff
@@ -124,17 +125,11 @@ def shape_from_bounding_box(bounding_box):
 def calc_gwcs_pixmap(in_wcs, out_wcs, shape=None):
     """ Return a pixel grid map from input frame to output frame.
     """
-    if shape:
-        bb = wcs_bbox_from_shape(shape)
-        log.debug("Bounding box from data shape: {}".format(bb))
-    else:
-        bb = in_wcs.bounding_box
-        log.debug("Bounding box from WCS: {}".format(in_wcs.bounding_box))
-
-    grid = gwcs.wcstools.grid_from_bounding_box(bb)
-    pixmap = np.dstack(reproject(in_wcs, out_wcs)(grid[0], grid[1]))
+    if shape and not np.array_equiv(shape, in_wcs.array_shape):
```

Comment:
again this isn't your fault as you didn't modify this, but it's best practice to use `is` or `is not` when comparing against `None`; see https://docs.astral.sh/ruff/rules/none-comparison/

Reply:
I do not find this harmful but I will update.